appyters/PrismEXP/PrismEXP.ipynb

```diff
@@ -0,0 +1,522 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%appyter init\n",
+    "import os, sys; sys.path.insert(0, os.path.realpath('..'))\n",
```

Comment:
FYI, this is only relevant for the example appyter because it runs in the appyter directory and is using the source code for appyter instead of the appyter module. If you have appyter installed with pip however this is irrelevant and just loading the parent directory into PYTHONPATH. It could be removed.